src/mame/drivers/plygonet.cpp

```diff
@@ -1054,7 +1054,7 @@ void polygonet_state::plygonet(machine_config &config)
 	Z80(config, m_audiocpu, 8000000);
 	m_audiocpu->set_addrmap(AS_PROGRAM, &polygonet_state::sound_map);
 
-	config.set_perfect_quantum(m_maincpu); // TODO: TEMPORARY!  UNTIL A MORE LOCALIZED SYNC CAN BE MADE
+	config.set_maximum_quantum(attotime::from_hz(600)); // NOTE: This does not appear to be necessary, but is retained for later testing.
```

Comment:
I know this is arbitrary but it should really be a direct correlation to the interested CPU clocks available and 600 Hz definitely isn't. MAME scheduler is known to miss a sync on long runs otherwise, cfr. Ninja Warriors.

Reply:
Your feedback is unwelcome.

Reply:
MAME’s always running with a fixed 50Hz periodic timer for sound updates which is unrelated to CPU frequencies. I don’t see how what amounts to another fixed timer is going to cause bigger problems.

Reply:
That’s what I’ve asked to myself when fixing final stage missing enemies in ninjaw. Gotta eventually love bugs that happens on prolonged sessions here.